raft/raft.go

```diff
@@ -715,7 +714,6 @@ func (r *raft) Step(m pb.Message) error {
 					r.id, r.raftLog.lastTerm(), r.raftLog.lastIndex(), r.Vote, m.Type, m.From, m.LogTerm, m.Index, r.Term, r.electionTimeout-r.electionElapsed)
 				return nil
 			}
-			lead = None
```

Comment:
why this is removed?

Reply:
Seems like it was removed because it is no longer used, based on that change below